.github/workflows/evm.yml

```diff
@@ -113,7 +113,7 @@ jobs:
           RUSTDOCFLAGS: "-D rustdoc::broken-intra-doc-links -D rustdoc::private_intra_doc_links"
 
       # FIXME: Calculate coverage and define how to upload all template coverage reports
```

Comment:
I would include a comment explaining why it was disabled(for whoever checks this)